src/components/item/sharing/ConfirmMembership.tsx

```diff
@@ -41,26 +43,52 @@ const DeleteItemDialog = ({
 
   const onDelete = () => {
     if (membershipToDelete?.id) {
-      deleteItemMembership({ id: membershipToDelete?.id, itemId: item.id });
+      deleteItemMembership({ id: membershipToDelete.id, itemId: item.id });
       handleClose();
     }
   };
 
   let dialogText = '';
+  const isDeletingLastAdmin =
+    hasOnlyOneAdmin && membershipToDelete?.permission === PermissionLevel.Admin;
   // incase of deleting the only admin
-  if (
-    hasOnlyOneAdmin &&
-    membershipToDelete?.permission === PermissionLevel.Admin
-  ) {
-    dialogText = translateBuilder(BUILDER.DELETE_ONLY_ADMIN_ALERT_MESSAGE);
+  if (isDeletingLastAdmin) {
+    dialogText = translateBuilder(BUILDER.DELETE_LAST_ADMIN_ALERT_MESSAGE);
   } else if (member?.id === membershipToDelete?.member?.id) {
     // deleting yourself
     dialogText = translateBuilder(BUILDER.DELETE_OWN_MEMBERSHIP_MESSAGE);
   } else {
     // delete other members
-    dialogText = translateBuilder(BUILDER.DELETE_MEMBERSHIP_MESSAGE);
+    dialogText = translateBuilder(BUILDER.DELETE_MEMBERSHIP_MESSAGE, {
+      name: membershipToDelete?.member.name,
+      permissionLevel: membershipToDelete?.permission,
+    });
   }
-  return (
+  return isDeletingLastAdmin ? (
+    <Snackbar
+      open={open}
+      autoHideDuration={6000}
+      onClose={handleClose}
+      anchorOrigin={{ vertical: 'top', horizontal: 'center' }}
+    >
+      <Alert
+        severity="error"
+        action={
+          <IconButton
+            aria-label="close"
+            color="inherit"
+            size="small"
+            onClick={handleClose}
+          >
+            <CloseIcon fontSize="inherit" />
+          </IconButton>
+        }
+      >
+        <AlertTitle>Error</AlertTitle>
+        {dialogText}
+      </Alert>
+    </Snackbar>
```

Comment:
Ah ! I did not see it like that but I guess it also works. 

Maybe if we can try to keep it consistent with the other dialogs 😅  I was thinking of putting the Alert inside the Dialog. I think it will be better to still open the dialog as this focuses the attention of the user. I feel that a Snackbar/toast is maybe a bit too small and could be out of sight on larger screens. 

What do you think ?

Reply:
I think it will be a bit strange/odd to have an alert within the dialog itself!

I feel like I'm not getting the point, would u please confirm that this is what we want to have?

![image](https://github.com/graasp/graasp-builder/assets/49619087/54034107-10ba-4b95-ac68-71dc28f51b98)